.github/workflows/ci-orchestrator.yml

```diff
@@ -19,19 +19,36 @@ jobs:
 
   ci_docker:
     needs: changes
-    if: ${{ needs.changes.outputs.docker == 'true' }}
-    uses: ./.github/workflows/python-build-and-push-docker-image.yml
+    uses: Energinet-DataHub/.github/.github/workflows/python-build-and-push-docker-image.yml@v12
+    with:
+      docker_changed: ${{ needs.changes.outputs.docker == 'true' }}
+      docker_changed_in_commit: ${{ needs.changes.outputs.docker_in_commit == 'true' }}
 
   create_release:
-    needs: [changes]
+    needs: changes
     if: ${{ needs.changes.outputs.telemetry == 'true' || needs.changes.outputs.spark-sql-migrations == 'true' }}
     uses: ./.github/workflows/create-release-tag.yml
     secrets: inherit
 
+  #
+  # Run Python  unit tests
```

Comment:
```suggestion
  # Run Python unit tests
```

Reply:
double new line but no double space between words... this is tough...
fixed it :d